robottelo/ui/computeresource.py

```diff
@@ -71,9 +71,7 @@ def _configure_resource_provider(
                     'button'
                 ))
                 self.click(locators[button_locator])
-                self.find_element(
-                    locators[param_locator]
-                ).select_by_visible_text(parameter_value)
+                self.select(locators[param_locator], parameter_value)
```

Comment:
We do not use `select` method directly anymore if we can, but `assign_value` instead